Common/Product/SharedProject/Utilities.cs

```diff
@@ -344,7 +332,7 @@ public static class Utilities {
                     unknown1 = QueryInterfaceIUnknown(obj1);
                     unknown2 = QueryInterfaceIUnknown(obj2);
 
-                    isSame = IntPtr.Equals(unknown1, unknown2);
+                    isSame = Equals(unknown1, unknown2);
```

Comment:
Is that right? Which `Equals` is that going to call, this class' `Equals`? If unsure I'd leave it as is, unless you know it was causing a problem.

Reply:
OK. That was C# suggestion.